src/app/screens/LNURLPay/index.tsx

```diff
@@ -263,53 +270,25 @@ function LNURLPay() {
     return [];
   }
 
-  function renderSuccessAction() {
-    if (!successAction) return;
-    let descriptionList: [string, string | React.ReactNode][] = [];
-    if (successAction.tag === "url") {
-      descriptionList = [
-        ["Description", successAction.description],
-        [
-          "Url",
-          <>
-            {successAction.url}
-            <div className="mt-4">
-              <Button
-                onClick={() => {
-                  if (successAction.url) utils.openUrl(successAction.url);
-                }}
-                label="Open"
-                primary
-              />
-            </div>
-          </>,
-        ],
-      ];
-    } else if (successAction.tag === "message") {
-      descriptionList = [["Message", successAction.message]];
+  function renderResultAction() {
+    if (!result) return;
+
+    if (result === "success" && !successMessage) {
+      setSuccessMessage(
+        `${valueSat} (SATS) ${showFiat ? "(" + fiatValue + ")" : ""} ${t(
+          "were_sent_to"
+        )} ${navState.origin?.name || getRecipient()}`
+      );
```

Comment:
Not sure what to do here regarding the message to show exactly.
//cc @escapedcat @bumi

Reply:
Hm, what do you mean? Should we have a look together?